GloVe/download_glove_840b_300d.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo "Downloading GloVe Common Crawl (840B tokens, 2.2M vocab, cased, 300d vectors, 2.03 GB download)"
+wget http://nlp.stanford.edu/data/glove.840B.300d.zip
```

Comment:
Actually I realize `md5sum` is not on Mac. Let me update this to be something that's cross-platform.